test/k8s/simple_test.go

```diff
@@ -73,17 +73,21 @@ data:
 			Expect(err).ToNot(HaveOccurred(), "invalid value of TEST_NUM_SERVICES")
 			numServices = i
 		}
-		services := graph.GenerateRandomServiceMesh(time.Now().Unix(), numServices, 50, 1, 1)
+
+		loc, err := time.LoadLocation("America/New_York")
+		Expect(err).ToNot(HaveOccurred())
+		seed := time.Date(1997, time.August, 29, 2, 14, 0, 0, loc).Unix()
+
+		services := graph.GenerateRandomServiceMesh(seed, numServices, 50, 1, 1)
```

Comment:
this is just an int64. I though you would put something random like "1337", but this parsing date works I guess

Reply:
true... a straight int is indeed easier